src/pineko/check.py

```diff
@@ -140,66 +141,47 @@ def is_fonll_b(fns, lumi):
     return False
 
 
-def get_order_list(grid, max_as, max_al):
-    """Return the order list after filtering according to max_as and max_al."""
+def orders(grid, max_as, max_al) -> list:
+    """Select the relevant orders.
+
+    The orders in the grid are filtered according to `max_as` and `max_al`.
+
+    """
     order_array = np.array([order.as_tuple() for order in grid.orders()])
     order_mask = pineappl.grid.Order.create_mask(grid.orders(), max_as, max_al, True)
     order_list = order_array[order_mask]
     return order_list
 
 
-def pure_qcd_orders(order_list):
-    """Return the pure alpha_s orders appearing in the grid."""
+def pure_qcd(orders):
+    """Select the QCD LO and pure QCD corrections to it."""
     as_orders = []
```

Comment:
Now dead :)

Reply:
?

Reply:
Ah you mean that `as_orders` is not used anymore, right?